app/src/main/java/org/kiwix/kiwixmobile/zim_manager/fileselect_view/ZimFileSelectFragment.java

```diff
@@ -204,8 +204,15 @@ public boolean onActionItemClicked(ActionMode mode, MenuItem item) {
                */
 
               File shareFile = new File(shareFilePath);
-              Uri shareContentUri = FileProvider.getUriForFile(getContext(),
-                  BuildConfig.APPLICATION_ID + ".fileprovider", shareFile);
+              Uri shareContentUri;
+              if(Build.VERSION.SDK_INT >=24) {
+                shareContentUri = FileProvider.getUriForFile(getContext(),
+                    BuildConfig.APPLICATION_ID + ".fileprovider", shareFile);
+              }
+              else {
```

Comment:
`else` should be in the same line as the preceding curly bracket. You must format your code using `Ctrl + Shift + L` before committing to avoid such mistakes. Also, for small changes there should a single commit per PR you can squash the commits and force push the changes.